std/algorithm/sorting.d

```diff
@@ -2308,6 +2308,13 @@ unittest
     topN(a, b);
     sort(a);
     assert(a == [0, 1, 2, 2, 3]);
+
+    // bug 15420
```

Comment:
Bug tests should be in their own `unittest` block and should not show up in the docs.